api/src/domain/license-status/searchLicenseStatusFactory.test.ts

```diff
@@ -97,13 +111,67 @@ describe("searchLicenseStatus", () => {
     });
 
     const result = await searchLicenseStatus(nameAndAddress, "Home improvement");
-    expect(result.checklistItems).toEqual([
-      {
-        title: "Item 2",
-        status: "ACTIVE",
-      },
+    expect(result.status).toEqual("EXPIRED");
+    expect(result.checklistItems).toEqual(
+      expect.arrayContaining([
+        {
+          title: "Item 1",
+          status: "ACTIVE",
+        },
+        {
+          title: "Item 2",
+          status: "ACTIVE",
+        },
+      ])
+    ); 
+  } );
+
+  it("returns a license if it doesn't have an issue date", async () => {
```

Comment:
In the vein of "tests as documentation of what the code does", I think a test that is more along the lines of explicitly "it uses dateThisStatus as the date of an entity if issueDate is undefined, and it uses expirationDate as the date of an entity if the dateThisStatus is undefined" (maybe slightly less wordy and/or split into multiple tests if needed) but the idea is to show/construct a situation where the test _cannot_ pass unless that exact logic is coded.